### DISCUSSION TITLE
My view on DAO governance

### DISCUSSION THREAD

**Discussion by SasoLithops** [ringdao-community-guild/discussion#42] [Ideas]
I am involved and observe some good examples of DAO governance in Cosmos network, so I would like to share the knowledge I learned there and hopefully it sparks a disscusion to push DC DAO towards community governed and fully decentralised. LET IT BE KNOWN BY COMMUNITY I MEAN TEAM MEMBERS,AMBASSADORS AND EVERYONE INVESTED IN DARWINIA(people in TG, Discord, Reddit, Github....) 
I hope you join in discussion to make this better by sharing your view, experiences.... Enough for intro, let's go:-)

Here are key points I see:

- EVERY COMMUNITY MEMBER SHOULD BE ABLE TO VOTE

PRO: The voice and vote from everyone counts
         Even the ones with not much voting power can make proposal as they are equal to us all

CON: There could be ill proposals to steal treasury or your funds as everyone can post proposal.
         
Solution I saw:

**Osmosis made that before any proposal goes live for voting it must be discussed for at least 3 days on their offical disscusion forum.**
        https://forum.osmosis.zone/  
Just to get idea here is a great example of some of us discussing proposal on the same forum but for ION DAO a proposal before it will go live: https://forum.osmosis.zone/t/discussions-and-new-ideas-improve-ion-dao-governance-participation-rate/251

**There must be minimum requirement of RING set by the one giving proposal and another minimum ammount to make it go live.**
For example, this is up to discussion:
We could have that minimum ammount for creating proposal is 100 000 RING. For making that proposal go live 2 000 000 RING is needed. That extra 1 900 000 RING can be provided from anyone from community if they support the idea. After proposal goes live and voting period passes all RING is returned. UNLESS proposal gets mayority votes NO WITH VETO then RING is either burned or sen to treasury(I like the second version better)

- NOT HAVING VALIDATOR SET

Cosmos has one of the best decentralised validator set out there and it is now implementing shared security. Polkadot is way ahead there with parachains. However Darwinia not having validators but collators now which could provide most of the votes and take care of ill proposals this is an issue. I don't have much trust in our collators as first I don't know any(they don't talk with us) and they play with fees in not transparent way.  

So let's play with idea that no collators can vote with their joined power only their individual power.
We have similar example with ION DAO. You can see on link below how easily we reject ill minded proposals (last two)
https://ion.wtf/governance
We came to solution there that only staked ION counts for voting. It could be same here only staked Ring counts for voting, I would also add the higher the locking period higher the votes. Then we need to set quorum that needs to be reached for proposal to be valid. That is the hardest part. We set 30% in ION DAO. We will need to try and see what that % is in DCDAO.
Here I see huge reserve for our KTON. I thought we could come up with great ideas with KTON SubDAO to help DCDAO but maybe later, when more people show interest....

What do you all think? Every community member welcome to join in debate. Let's make it good;-)

**Comment by Laki389** [comment]
Before I get into the discussion I would like to be sure if you are distinguishing between Darwinia Governance and DCDAO Governance?

Because in Darwinia governance, anyone who has a Ring can propose and vote in referenda.

**quote 1** "Any changes, whether they involve adjusting pallet parameters or facilitating asset transfers, must now be executed through on-chain governance. This represents a significant leap towards establishing a truly decentralized platform for the Darwinia network." **end of quote 1**

**quote 2** "Public referenda

Any token holder in the system has the ability to propose and vote on public referenda by providing a bond. Once a proposal is submitted, others can show their support by endorsing it and staking tokens equivalent to the initial bond. During each launch period, the most endorsed proposal will be moved to the public referenda table, where it becomes eligible for voting. If a referenda is approved by the majority of token holders, it will be executed at a later stage; otherwise, it will be rejected. " **end of quote 2**

Links:

- Governance [https://docs.darwinia.network/governance-22e8b46c937749b6848e7a99e11fe81a](https://docs.darwinia.network/governance-22e8b46c937749b6848e7a99e11fe81a)

- Participate in Democracy [https://docs.darwinia.network/participate-in-democracy-663e7e9388454f1698663d6c8a4ce59c](https://docs.darwinia.network/participate-in-democracy-663e7e9388454f1698663d6c8a4ce59c)

- Participate in Council [https://docs.darwinia.network/participate-in-council-c816ba48ccc24f95b3ae6840d01e81e6](https://docs.darwinia.network/participate-in-council-c816ba48ccc24f95b3ae6840d01e81e6)

- Participate in Treasury [https://docs.darwinia.network/participate-in-treasury-daf3409d150c4db0a6b454da1108ba3c](https://docs.darwinia.network/participate-in-treasury-daf3409d150c4db0a6b454da1108ba3c)

**Reply by SasoLithops** [reply]
Like the title said I only focus on DAOs consequently DCDAO😉 But thank you for posting this anyways.

**Reply by Laki389** [reply]
> Like the title said I only focus on DAOs consequently DCDAO😉 But thank you for posting this anyways.

Alright.
I asked because on the discussion thread at https://github.com/orgs/dcdao/discussions/38 you write, among other things, that every Darwinia/Crab tokenomics change should be voted on by DCDAO governance.
So I would like to ask why there is a need for duplication of governance?
On Darwinia governance, anyone can propose and vote on referenda, that referenda can be for changing Darwinia/Crab tokenomics or for something else.

**Comment by SasoLithops** [comment]
To make it clear. We have to strongly distinguish between DARWINIA and DCDAO.

I think when making a proposal in Darwinia directly can cause some issues regarding where to discuss, how to alert community, learning what proposal is all about, getting feedback from proposer. I found that in polkadot very confusing. I wish I could show you how easy and transparent it is in Cosmos ecosystem where every proposal has code and written subject on what it is about, then opinions on that proposal from validators that voted with their vote and link to discussion.

What I asked in https://github.com/orgs/dcdao/discussions/38 was what you all think should be voted on by DCDAO. It was me trying to learn what you all think we should vote on as DCDAO.

My understanding is we as DCDAO are a seperate entity supporting Darwinia. And we will have our own governance. And when majority of us vote YES on something we can publish that proposal on Darwinia. Correct me if I am wrong. 

Now I have some experience in DAOs from Cosmos and Polkadot ecosystem. And wanted to share what imho is the governance solution that showed it actually works for DAOs. 

I also feel that we as DCDAO should have a seperate page/forum where we can disscuss proposals and to ask proposer what we don't understand or disagree.
I gave this as an example: https://forum.osmosis.zone/t/discussions-and-new-ideas-improve-ion-dao-governance-participation-rate/251

**Reply by Laki389** [reply]
> I think when making a proposal in Darwinia directly can cause some issues regarding where to discuss, how to alert community, learning what proposal is all about, getting feedback from proposer. I found that in polkadot very confusing. I wish I could show you how easy and transparent it is in Cosmos ecosystem where every proposal has code and written subject on what it is about, then opinions on that proposal from validators that voted with their vote and link to discussion.

Currently, as with DCDAO, Darwinia discussions are on Github, in the discussions section at https://github.com/darwinia-network/darwinia/discussions

- RFC: Proposal for Economic Simplification and Subitizing of the Crab Chain [https://github.com/darwinia-network/darwinia/discussions/1273](https://github.com/darwinia-network/darwinia/discussions/1273)

- Key points of next gov security and voting power. [https://github.com/darwinia-network/darwinia/discussions/1284](https://github.com/darwinia-network/darwinia/discussions/1284)

- RFC: Remove dispatch calls of update commission without chill and remove collator MaxCommission limitation [https://github.com/darwinia-network/darwinia/discussions/1272](https://github.com/darwinia-network/darwinia/discussions/1272)

etc.

> My understanding is we as DCDAO are a seperate entity supporting Darwinia. And we will have our own governance. And when majority of us vote YES on something we can publish that proposal on Darwinia. Correct me if I am wrong.

There is already governance, right? At https://snapshot.org/#/darwiniacommunitydao.eth
Or do you suggest elsewhere/another way?

> I also feel that we as DCDAO should have a seperate page/forum where we can disscuss proposals and to ask proposer what we don't understand or disagree. I gave this as an example: https://forum.osmosis.zone/t/discussions-and-new-ideas-improve-ion-dao-governance-participation-rate/251

I agree, the site looks great and is very user friendly unlike Github. Github is more for developers and if I were to draw a parallel with polkadot ecosystem wallets, Github is the same as polkadot.js for wallets.
Do you have  information on how much it costs to create a website like Osmosis have and maintain it?

**Reply by SasoLithops** [reply]
I agree with you on Github and parallel with polkadot.js. It is easy to create account but quite complicated from there on. That is why I propose seperate forum. 

I don't know cost for Osmosis, but I can ask if you would like to know. We have nowhere near that active community and no validators to have it similar. Was really just example about how to be more transparent about governance issues. 
That said I do own a small bussines and also have a webpage. Cost of owning domain and what goes with it for me is 18€ per year. I do changes to website myself it is not that hard. We would however need developers help to make it more proffesional...Since that is technical part, I spoke about that with our Nada on KTON DAO waiting for his reply.

Also to mention now you have many sites when this can be run cost free. I know a friend of mine she runs it with 0 cost. Also I don't mind if we continue having voting part on snapshot site. 

Much appreciate your points as always, but I feel we are moving away from  main point of this discussion. In this thread I was more focusing to discuss DAO governance from community member perspective and how they see DAO run it.

**Comment by Cindy884** [comment]
Thanks, Saso, I am thrilled to hear about your idea. 
After reading the discussion, I noticed two key points, democracy and safety. 
In terms of democracy, we should improve our member's activities and let them know what happens every day in Darwinia first. I will work with David to do this in the next three months. 
Safety can be divided into two parts, treasury safety and the other is governance safety. 
We can use the multi-sig wallet to solve the former. In DCDAO, we use SBT to ensure the proposal suits our vision. 
But I agree with your suggestions, we need to think about how to avoid malicious proposals in the future.
I am just afraid if people have to stake the Ring when they vote, it will harm to efficiency.

**Comment by SasoLithops** [comment]
Hi Cindy!

Glad to post what I learned. I hope David does as well so more good examples are shown here in this thread. 
I absolutely agree regarding your point on  security and safety. 
Sorry but I don't understand your last sentence. Could you please explain what you mean by "it will harm to efficency".

Imho I think only commited members like stakers and if possible definitely also liquidity providers who risk by taking Ring off the market should participate in DCDAO governance. But up to discussion if duration of staking or anything else in that matter should increase voting power. I haven't got to a final answer on that and still disagree with myself🤦😉

**Comment by SasoLithops** [comment]
It was very good disscussion about DAOs in our Darwinia Unofficial TG chat and GreenChiles made great point:

DAOs do end up being dominated by insiders and wealthy stakeholders, and really become decentralized in name only.

So we do need better solution in governance that wouldn't lead to this. 

Maybe solution is not to let only staked Ring give the voting power.
